cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -880,6 +885,29 @@ protected int resumeSchedule(String namespace, String appName, String schedule)
     return doGetSchedules(namespace, schedulesUrl);
   }
 
+  protected List<ScheduleDetail> listSchedulesByTriggerProgram(String namespace, ProgramId programId,
+                                                               co.cask.cdap.api.ProgramStatus... programStatuses)
+    throws Exception {
+    String schedulesUrl = String.format("schedules/trigger-type/program-status?trigger-namespace-id=%s" +
+                                          "&trigger-app-name=%s&trigger-app-version=%s" +
+                                          "&trigger-program-type=%s&trigger-program-name=%s", programId.getNamespace(),
+                                        programId.getApplication(), programId.getVersion(),
+                                        programId.getType().getCategoryName(), programId.getProgram());
+    if (programStatuses.length > 0) {
+      List<co.cask.cdap.api.ProgramStatus> programStatusList = Arrays.asList(programStatuses);
```

Comment:
Doesn't `Lists.transform` return a list? It doesn't seem to be used anywhere. Is the result of `Lists.transform` supposed to be used on line 906?